pkgs/desktops/lomiri/development/lomiri-app-launch/default.nix

```
@@ -96,8 +98,8 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   cmakeFlags = [
-    "-DENABLE_MIRCLIENT=OFF"
-    "-DENABLE_TESTS=${lib.boolToString finalAttrs.doCheck}"
+    (lib.cmakeBool "ENABLE_MIRCLIENT" false)
+    (lib.cmakeBool "ENABLE_TESTS" finalAttrs.doCheck)
```

```suggestion
    (lib.cmakeBool "ENABLE_TESTS" finalAttrs.finalPackage.doCheck)
```

Technically mkDerivation still overrides doCheck, so check final package attrs after the mkDerivation call instead of before